### DISCUSSION TITLE
SP API - POST_PRODUCT_PRICING_DATA - Unexpected Feed Error

### DISCUSSION THREAD

**Discussion by cmiles** [amzn/selling-partner-api-models#3679] [General]
I am trying to submit an XML POST_PRODUCT_PRICING_DATA feed to the SP API and getting an unexpected error:

ResultDescription=12629 > 0.00 price (standard or sales) will not be accepted. Please ensure that every SKU in your feed has a price at least equal to or greater than 0.01

The error message seems clear enough but I'm having trouble how it relates to the submitted feed file - pasted in below - which seems to provide a valid Standard Price.

```
<?xml version="1.0" encoding="utf-8"?>
<AmazonEnvelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
  <Header>
    <DocumentVersion>1.02</DocumentVersion>
    <MerchantIdentifier>ZZZZZZZZZZZZZZ</MerchantIdentifier>
  </Header>
  <MessageType>Price</MessageType>
  <Message>
    <MessageID>1</MessageID>
    <OperationType>Update</OperationType>
    <Price>
      <SKU>12629</SKU>
      <StandardPrice currency="USD">134.95</StandardPrice>
      <MinimumSellerAllowedPrice currency="USD">134.94</MinimumSellerAllowedPrice>
      <MaximumSellerAllowedPrice currency="USD">134.96</MaximumSellerAllowedPrice>
    </Price>
  </Message>
</AmazonEnvelope>
```

I have verified the SKU exists and we have used the SP-API for a number of activities including retrieving reports, retrieving Orders, submitting Order Acknowledgments, Submitting POST_INVENTORY_AVAILABILITY_DATA and POST_PRODUCT_DATA XML Feeds.

With other successful API interactions and the error message I'm getting it seems that there is a problem with the Message.

I've tried a number of variations of the XML above without success - without the min/max pricing for example but get the same error.

I'm hoping someone can spot and point me at the error - or perhaps suggest some additional troubleshooting steps!

We don't appear to be having trouble with this in our older MWS code.

Thanks for any help! Appreciated!!
Charles

**Comment by Moerty** [comment]
Found a solution?

**Comment by paul-van-der-hoeven** [comment]
We experienced the same issue.  
We noticed that submitting a price change once would process successfully.
Sending the exact same change within a few minutes would produce this error.
I realize that this is not a solution or even a work-around, but at least be assured that your code is not the problem.
Amazon needs to simply accept the change, or provide a meaningful response.
